encrypted-media-respec.html

```diff
@@ -1542,17 +1542,27 @@
                     <li><p>Let <var>session id</var> be a unique <a def-id="session-id"></a> string.</p>
                       <p>If the result of running the <a def-id="is-persistent-session-type-algorithm"></a> algorithm on <var>session type</var> is <code>true</code>, the ID MUST be unique within the <a def-id="origin"></a> of this object's <a def-id="document-concept"></a> over time, including across Documents and browsing sessions.</p>
                     </li>
-                    <li><p>Let <var>message</var> be a license request for the <var>requested license type</var> generated based on the <var>sanitized init data</var>, which is interpreted per <var>initDataType</var>.</p>
+                    <li><p>Let <var>message</var> be a license request for the <var>requested license type</var> generated based on the <var>sanitized init data</var> interpreted per <var>initDataType</var>, or an individualization request.</p>
```

Comment:
I'm open to other solutions as well. I like that this one clearly breaks out the two cases (request generated and not) and allows the `update()` requirement to be in the second block.